src/main/java/io/vertx/core/spi/logging/package-info.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2011-2018 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+
+/**
+ * @deprecated see https://github.com/eclipse-vertx/vert.x/issues/2774
+ */
+@Deprecated
```

Comment:
I don't think we need to deprecate this as we should continue using an internal logger after v4

Reply:
Right, we will continue to have an internal logging API. But I believe it would be better to move it to an `impl` or `internal` package. We can do the changes in the module stack. People still using this API  after Vert.x 4 will have be warned.

Reply:
this is an SPI package, SPI are not intended to be used directly, so it's fine where it stands.

Reply:
Right, sorry, I overlooked your comment and thought this was about the other package.

Reply:
I've updated the PR and rebased on master